### DISCUSSION TITLE
Noob question #1: can I have a different /admin access address?

### DISCUSSION THREAD

**Discussion by LuizCesarLeite** [strapi/strapi#6949] [Legacy Topics]
Hi there!

I discovered Strapi some days and I'm pretty excited to know more about it!

My first question:

I already have in my domain an application that uses /admin. Can I change to something like "mydomain.com/strapiadmin" ?

Thanks a lot!

**Comment by iagocalazans** [comment]
Yes! You just need to insert the new path in config/environments/{the_one_you_want}/server.json. Insert it into the block code:

```
"admin": {
    "autoOpen": false
  }
```
The domain you want as your path:
```
"admin": {
    "autoOpen": false,
    "path": "/myadmindomain"
  }
```

**Reply by LuizCesarLeite** [reply]
Cool, Iago! Thanks a lot!

I'll try it in a few moments :-)

**Reply by derrickmehaffy** [reply]
It's not longer path, it's `url`: https://strapi.io/documentation/v3.x/concepts/configurations.html#server

**Comment by LuizCesarLeite** [comment]
@iagocalazans 

I dig into /config in my project's directory but I didn't found the /environment directory.

Is /config inside it?

**Reply by derrickmehaffy** [reply]
You are looking at the beta docs (older) not the stable docs: https://strapi.io/documentation/v3.x/concepts/configurations.html#environments

**Comment by LuizCesarLeite** [comment]
Thanks, @derrickmehaffy !

There's a lot of info to process :-)

I'll dig into it and came back later!

Thank you!

**Reply by derrickmehaffy** [reply]
No problem 😋
